src/components/Shared/NavFooter.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import {
+  StyledNavFooter,
+  StyledNavFooterButtons,
+} from "../../styles/NavFooter.css.js";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faHome,
+  faTruck,
+  faStreetView,
+  faWrench,
+} from "@fortawesome/free-solid-svg-icons";
+
+const navPages = [
+  {
+    title: "Home",
+    link: "/",
+    icon: faHome,
+  },
+  {
+    title: "My Work Orders",
+    link: "/my-work-orders",
+    icon: faStreetView,
+  },
+  {
+    title: "All Work Orders",
+    link: "/all-work-orders",
+    icon: faTruck,
+  },
+  {
+    title: "New Work Order",
+    link: "/work-order/new",
+    icon: faWrench,
+  },
+];
+
+const linkColors = {
+  inactive: "#6C757C",
+  active: "#307BFF",
```

Comment:
not necessary right now, but we could put all the colors we use in a constants file.

Reply:
Yep - will make any color/themes changes easier in the long run.